io.github.zapzap-linux.zapzap.yaml

```diff
@@ -0,0 +1,55 @@
+app-id: io.github.zapzap-linux.zapzap
+runtime: org.kde.Platform
+runtime-version: "6.6"
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: "6.6"
+command: zapzap
+finish-args:
+  - "--share=ipc"
+  - "--socket=wayland"
+  - "--socket=fallback-x11"
+  - "--socket=pulseaudio"
+  - "--share=network"
+  - "--device=all"
+  - "--device=dri"
+  - "--filesystem=host"
+  - "--filesystem=xdg-download"
+  - "--talk-name=org.freedesktop.Notifications"
+  - "--talk-name=org.kde.StatusNotifierWatcher"
+  - "--talk-name=com.canonical.AppMenu.Registrar"
+  - "--talk-name=com.canonical.Unity"
+  - "--talk-name=com.canonical.indicator.application"
+  - "--talk-name=org.ayatana.indicator.application"
+  - "--unset-env=XDG_DATA_HOME"
```

Comment:
why is this unset?

Reply:
Honestly, I don't understand much about Flatpak's permissions. I've always used these because I asked for help from colleagues from the community and so I use it for over a year.

 I will spend time researching better to avoid future problems. Pardon me